### ISSUE TITLE
About Lion optimizer

### ISSUE THREAD

**Issue by coni-coco** [baaivision/EVA#75]
Can you provide detailed Lion setting(learning rate and weight decay) in Table 18(EVA02 paper)?
